torch/utils/hipify/cuda_to_hip_mappings.py

```diff
@@ -7302,6 +7302,7 @@
         ("CUBLASLT_EPILOGUE_GELU_BIAS", ("HIPBLASLT_EPILOGUE_GELU_BIAS", CONV_MATH_FUNC, API_BLAS)),
         ("cublasLtHandle_t", ("hipblasLtHandle_t", CONV_MATH_FUNC, API_BLAS)),
         ("cublasLtMatmulDesc_t", ("hipblasLtMatmulDesc_t", CONV_MATH_FUNC, API_BLAS)),
+        ("cublasLtMatrixLayout", ("hipblasLtMatrixLayout_t", CONV_MATH_FUNC, API_BLAS)),
```

Comment:
Is this required?
cublasLtMatrixLayout_t is type is already handled.

Reply:
removed duplicate